src/client.ts

```diff
@@ -182,10 +187,12 @@ export class Client {
       params.filter
     )
     this.consumers.set(consumerId, { connection, consumer, params })
-    await this.declareConsumerOnConnection(params, consumerId, this.connection)
-    this.logger.info(
-      `New consumer created with stream name ${params.stream}, consumer id ${consumerId} and offset ${params.offset.type}`
-    )
+    try {
+      await this.declareConsumerOnConnection(params, consumerId, this.connection)
+      this.logger.info(
+        `New consumer created with stream name ${params.stream}, consumer id ${consumerId}, connection index ${this.currentConsumerConnectionIndex} and offset ${params.offset.type}`
+      )
+    } catch (error) {}
```

Comment:
Is it wise to catch the error here and do nothing? What happens if we fail to declare the consumer on the connection?